runner/src/indexer/indexer.ts

```diff
@@ -219,83 +222,87 @@ export default class Indexer {
-            await this.writeLog(`context.db.select_${sanitizedTableName}`, blockHeight,
-              `Calling context.db.select_${sanitizedTableName}.`,
-              `Selecting objects with values ${JSON.stringify(object)} in table ${tableName} on schema ${schemaName} with ${limit === null ? 'no' : limit} limit`);
-            dmlHandler = dmlHandler ?? await this.deps.DmlHandler.create(account);
-            return await dmlHandler.select(schemaName, tableName, object, limit);
-          },
-          [`update_${sanitizedTableName}`]: async (whereObj: any, updateObj: any) => {
-            await this.writeLog(`context.db.update_${sanitizedTableName}`, blockHeight,
-              `Calling context.db.update_${sanitizedTableName}.`,
-              `Updating objects that match ${JSON.stringify(whereObj)} with values ${JSON.stringify(updateObj)} in table ${tableName} on schema ${schemaName}`);
-            dmlHandler = dmlHandler ?? await this.deps.DmlHandler.create(account);
-            return await dmlHandler.update(schemaName, tableName, whereObj, updateObj);
-          },
-          [`upsert_${sanitizedTableName}`]: async (objects: any, conflictColumns: string[], updateColumns: string[]) => {
-            await this.writeLog(`context.db.upsert_${sanitizedTableName}`, blockHeight,
-              `Calling context.db.upsert_${sanitizedTableName}.`,
-              `Inserting objects with values ${JSON.stringify(objects)} in table ${tableName} on schema ${schemaName}. Conflict on columns ${conflictColumns.join(', ')} will update values in columns ${updateColumns.join(', ')}`);
-            dmlHandler = dmlHandler ?? await this.deps.DmlHandler.create(account);
-            return await dmlHandler.upsert(schemaName, tableName, Array.isArray(objects) ? objects : [objects], conflictColumns, updateColumns);
-          },
-          [`delete_${sanitizedTableName}`]: async (object: any) => {
-            await this.writeLog(`context.db.delete_${sanitizedTableName}`, blockHeight,
-              `Calling context.db.delete_${sanitizedTableName}.`,
-              `Deleting objects with values ${JSON.stringify(object)} in table ${tableName} on schema ${schemaName}`);
-            dmlHandler = dmlHandler ?? await this.deps.DmlHandler.create(account);
-            return await dmlHandler.delete(schemaName, tableName, object);
+          [`${sanitizedTableName}`]: {
+            insert: async (objects: any) => {
+              await this.writeLog(`context.db.${sanitizedTableName}.insert`, blockHeight,
+                `Calling context.db.${sanitizedTableName}.insert.`,
+                `Inserting object ${JSON.stringify(objects)} into table ${tableName} on schema ${schemaName}`);
+              dmlHandler = dmlHandler ?? await this.deps.DmlHandler.create(account);
+              return await dmlHandler.insert(schemaName, tableName, Array.isArray(objects) ? objects : [objects]);
+            },
```

Comment:
```suggestion
            insert: async (objects: any) => {
              await this.writeLog(
                `context.db.${sanitizedTableName}.insert`,
                blockHeight,
                `Calling context.db.${sanitizedTableName}.insert.`,
                `Inserting object ${JSON.stringify(objects)} into table ${tableName} on schema ${schemaName}`
              );

              dmlHandler = dmlHandler ?? await this.deps.DmlHandler.create(account);
              return await dmlHandler.insert(schemaName, tableName, Array.isArray(objects) ? objects : [objects]);
            },
```
Nit: Can we please add some formatting here and below, it's quite dense and hard to read 😅

Reply:
Haha sorry. Will do!

Reply:
Let's enforce it in eslint? Not in this PR of course.